src/ssr/IsUserFellow.tsx

```diff
@@ -0,0 +1,29 @@
+import { prisma } from "src/server/db";
```

Comment:
why is this file in it's own `ssr` directory? 🤔

Reply:
In which dir should I keep this file?

Reply:
should be a util

Reply:
what does ssr represent?